Assets/Mirror/Runtime/NetworkManagerHUD.cs

```diff
@@ -31,9 +35,31 @@ public class NetworkManagerHUD : MonoBehaviour
         /// </summary>
         public int offsetY;
 
+        /// <summary>
+        /// Outputs the current devices IP to gui input, useful if your are host/server.
+        /// Only use the WAN check when necessary, as it uses an external service.
+        /// For further connection information, visit: https://mirror-networking.com/docs/Articles/FAQ.html#how-to-connect
+        /// </summary>
+        [Tooltip("localhost for games on same device, LAN for different devices same wifi, WAN for external connections. See summary for more information.")]
```

Comment:
"LAN for different devices on the same network" rather than Wifi, small thing but I couldn't help but notice. And I think it should be put out properly to avoid confusion of any kind.

Reply:
Yeah i was debating that too, will adjust.